applications/ExaquteSandboxApplication/custom_processes/divergence_process.h

```diff
@@ -0,0 +1,217 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+//
+
+#ifndef KRATOS_DIVERGENCE_PROCESS_H
+#define KRATOS_DIVERGENCE_PROCESS_H
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/geometry_utilities.h"
+#include "includes/model_part.h"
+#include "processes/process.h"
+#include "utilities/variable_utils.h"
+#include "exaqute_sandbox_application_variables.h"
+
+// Application includes
+
+namespace Kratos
+{
+    ///@addtogroup ExaquteSandboxApplication
+    ///@{
+
+    ///@name Kratos Globals
+    ///@{
+
+    ///@}
+    ///@name Type Definitions
+    ///@{
+
+    ///@}
+    ///@name  Enum's
+    ///@{
+
+    ///@}
+    ///@name  Functions
+    ///@{
+
+    ///@}
+    ///@name Kratos Classes
+    ///@{
+
+    /// Process to compute divergence
+    /**
+     * This process computes the divergence and the seminorm of the velocity field.
+     * We define VELOCITY_H1_SEMINORM as: \left \| \nabla u_{h} \right \|_{L^2(K)}^2 ,
+     * and DIVERGENCE as \left \| \nabla \cdot u_{h} \right \|_{L^2(K)}^2 ,
+     * where u is the velocity field and K an element of the domain \Omega.
+     */
+    class KRATOS_API(EXAQUTE_SANDBOX_APPLICATION) DivergenceProcess : public Process
+    {
+    public:
+        ///@name Type Definitions
+        ///@{
+
+        /// Pointer definition of Process
+        KRATOS_CLASS_POINTER_DEFINITION(DivergenceProcess);
+
+        ///@}
+        ///@name Life Cycle
+        ///@{
+
+        /// Default constructor
+        /**
+         * @brief Construct DivergenceProcess object
+         * @param rModelPart Model part the process is applied to
+         */
+        DivergenceProcess(
```

Comment:
I suggest to create an alternative model + parameters constructor. This would enable it's usage from the GUI.

Reply:
Done.